jqjq.jq

```diff
@@ -2737,14 +2751,7 @@ def jqjq($args; $env):
       ) as $builtins_env
     | _inputs
     | eval($opts.filter; {"$ENV": $env}; $builtins_env)
-    | if $opts.raw_output and type == "string" then
-        if $opts.raw_output0 and contains("\u0000") then
-          error("Cannot dump a string containing NUL with --raw-output0 option")
-        end
-      else _tojson($opts)
-      end
-    | if $opts.raw_no_lf | not then ., "\n" end
-    | if $opts.raw_output0 then ., "\u0000" end
+    | dump($opts)
```

Comment:
`_filter/1` looks quite neat now!

Reply:
It's just like how it was before color support! Just with `dump($opts)` instead of `tojson` haha

Reply:
Aha hehe, re-neated, good!